src/common/tests/btree.sh

```diff
@@ -96,7 +103,24 @@ run_test()
     printf "\nOptions: IPL='%s' IDIR='%s' PMEM='%s'\n" "$IPL" "$IDIR" "$PMEM"
     test_conf=$(gen_test_conf_string "${IPL}" "${PMEM}")
 
-    if [ -z ${PERF} ]; then
+    if [ "${RPC}" != "" ]; then
+        echo "B+tree RPC tracking emulation functional test..."
+        DAOS_DEBUG="$DDEBUG"                        \
+        eval "${VCMD[@]}" "$BTR" --start-test \
+        "btree RPC tracking emulation functional test" \
+        -C "o:$ORDER" \
+        -D
+
+        echo "B+tree batch operations test..."
+        eval "${VCMD[@]}" "$BTR" \
```

Comment:
(lint) eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string). [SC2294]